### PR TITLE
US16615: Message Timestamp

### PR DESCRIPTION
**Pull request by lukemoderwell**
## Problem
Currently series pages don't have timestamps on the message cards

## Solution
adding them timestamps

### Corresponding Branch
*Add link to crdschurch/repo-name-here#issue-number (if applicable). This helps the code reviewer know that corresponding work exists and where to find it.*

## Testing
*Include information on how to test your work here (if applicable).*

### CODE REVIEW

File: _includes/_message-card.html [hunk 1]
```diff
@@ -1,19 +1,26 @@
 {% if message.image.url %}
-  {% assign image = message.image.url %}
+{% assign image = message.image.url %}
```
Comment (seancdavis) [inline _includes/_message-card.html]: Why the de-indentation? Typically the best practice is to indent logic within a conditional. See some Liquid examples [here](https://shopify.github.io/liquid/basics/operators/).

File: _includes/_message-card.html [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
 {% else %}
-  {% assign image = page.image.url %}
+{% assign image = page.image.url %}
 {% endif %}
 
 <div class="card">
   <a class="relative block" href="{{ message.url }}" title="{{ message.title }}">
-    <img src="{{ image | imgix: site.imgix }}?{{ site.imgix_params.placeholder_sixteen_nine }}" sizes="{{ site.image_sizes.cards_2x }}" alt="{{ message.title }}" data-optimize-img>
+    {% if message.duration > 0 %}
+    <div class="card-stamp bg-charcoal text-white">
```
Comment (seancdavis) [inline _includes/_message-card.html]: Thinking you'll want the play icon as well, to be consistent with other implementation.

File: _includes/_message-card.html [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
+{% assign image = message.image.url %}
 {% else %}
-  {% assign image = page.image.url %}
+{% assign image = page.image.url %}
 {% endif %}
 
 <div class="card">
   <a class="relative block" href="{{ message.url }}" title="{{ message.title }}">
-    <img src="{{ image | imgix: site.imgix }}?{{ site.imgix_params.placeholder_sixteen_nine }}" sizes="{{ site.image_sizes.cards_2x }}" alt="{{ message.title }}" data-optimize-img>
+    {% if message.duration > 0 %}
```
Comment (seancdavis) [inline _includes/_message-card.html]: What happens if message is missing -- does this statement catch that condition as well?

File: _includes/_message-card.html (hunk 2 above)
Comment (lukemoderwell) [reply _includes/_message-card.html]: I did leave out the play icon on purpose. It felt less helpful on a page that will always be messages but I won't die on the no-icon hill.

File: _includes/_message-card.html (hunk 1 above)
Comment (lukemoderwell) [reply _includes/_message-card.html]: My stupid formatter does this stuff. It's annoying and I'm truly sorry.

Comment (j-brammer) [reply _includes/_message-card.html]: His auto formatter doesn't understand liquid and lines them up

File: _includes/_message-card.html (hunk 3 above)
Comment (lukemoderwell) [reply _includes/_message-card.html]: If no duration this label will not be rendered.

**Pull request merged**